app/battle/start.py

```diff
@@ -0,0 +1,16 @@
+from app.knight.creation import Knight

```

Comment:
The import statement should use double quotes for consistency with the code style guide. Change `from app.knight.creation import Knight` to `from app.knight.creation import "Knight"`.